tableauserverclient/models/site_item.py

```diff
@@ -226,7 +227,9 @@ def user_quota(self) -> Optional[int]:
 
     @user_quota.setter
     def user_quota(self, value: Optional[int]) -> None:
-        if any((self.tier_creator_capacity, self.tier_explorer_capacity, self.tier_viewer_capacity)):
+        if value is not None and any(
```

Comment:
seems kind of odd that the only way to do this is to go through a step where both user quota and site capacity are None.  Maybe a helper method like replace_capacity_with_quota(user_quota=) and vice versa would make this cleaner to do?